ast/builtins.go

```diff
@@ -2000,6 +2001,17 @@ var NetCIDRContainsMatches = &Builtin{
 	),
 }
 
+// NetCIDRMerge merges IP addresses and subnets into the smallest possible list of CIDRs.
+var NetCIDRMerge = &Builtin{
+	Name: "net.cidr_merge",
+	Decl: types.NewFunction(
+		types.Args(
+			types.NewArray(nil, types.NewAny(types.S)),
```

Comment:
Is there any reason we can't accept a set of strings as well?

Reply:
I was wondering about that too, seems like it would be easy enough to add later if anyone needed it too

Reply:
Updated the builtin to accept set of strings.